xclim/data/fr.json

```diff
@@ -891,5 +891,17 @@
     "abstract": "Précipitations moins l'évapotranspiration potentielle en tant que mesure approximative d'un bilan hydrique de surface, où l'évapotranspiration potentielle est calculée avec une méthode donnée.",
     "description": "Précipitations moins l'évapotranspiration potentielle en tant que mesure approximative d'un bilan hydrique de surface, où l'évapotranspiration potentielle est calculée avec la méthode {méthode}.",
     "long_name": "Bilan hydrique."
+  },
+  "ROLLING_DRYDAYS_EVENTS": {
+    "title": "Événements de périodes séches.",
+    "abstract": "Le nombre de périodes de n jours minimum pendant lesquelles les précipitations accumulées sont inférieures à un seuil donné, pour chaque période de rééchantillonnage.",
```

Comment:
See my comment on the n day nuance but I think this is technically more accurate (even if it becomes wordy):
Applies to `ROLLING_DRYDAYS_COUNT` as well I think
````
"abstract": "Le nombre de périodes de n jours minimum pendant lesquelles les précipitations accumulées sur n jours sont inférieures à un seuil donné, pour chaque période de rééchantillonnage."

"description": "Le nombre de périodes de {window} jours minimum pendant lesquelles les précipitations accumulées sur {window} jours sont inférieures à {thresh}, pour chaque période de rééchantillonnage."
````

Reply:
Do you mean that it is better to keep the wording "pour chaque période de rééchantillonnage" instead of "Fréquence {freq:f} des périodes sèches"?

Reply:
No. Sorry. "Fréquence {freq:f} des périodes sèches" looks good. My comment was wrt to the second inclusion of {window}   in descritpion in the `accumulées`  portion.  Idea is to indicate that even when an event is longer than `window` the sum is always performed on `n==window` days